compiler/catala_utils/message.ml

```diff
@@ -164,28 +164,117 @@ module Content = struct
   let of_string (s : string) : t =
     [MainMessage (fun ppf -> Format.pp_print_text ppf s)]
 
+  let basic_msg ppf target content =
+    Format.pp_open_vbox ppf 0;
+    Format.pp_print_list
+      ~pp_sep:(fun ppf () -> Format.fprintf ppf "@,@,")
+      (fun ppf -> function
+        | Position pos ->
+          Option.iter
+            (fun msg -> Format.fprintf ppf "@[<hov>%t@]@," msg)
+            pos.pos_message;
+          Pos.format_loc_text ppf pos.pos
+        | MainMessage msg ->
+          Format.fprintf ppf "@[<hov 2>[%t] %t@]" (pp_marker target) msg
+        | Outcome msg ->
+          Format.fprintf ppf "@[<hov>[%t]@ %t@]" (pp_marker target) msg
+        | Suggestion suggestions_list -> Suggestions.format ppf suggestions_list)
+      ppf content;
+    Format.pp_close_box ppf ();
+    Format.pp_print_newline ppf ()
```

Comment:
Why two different message formats (`basic` and `fancy`)?

Reply:
`basic` is still used for logs and debugs